Src/CANProcess.c

```diff
@@ -35,15 +35,38 @@
 *     Parameters (list data type, name, and comment one per line):
 *
 *     Global Dependents:
-*	  1. QueueHandle_t	q_pedalbox_msg;
+*	  1. ;
 *
 *     Function Description:
-*			To be called by CAN1_RX0_IRQHandler in order to queue
-*			received CAN messages to be processed by RXCANProcessTask
+*			To be called by HAL_CAN_IRQHandler when a CAN message is received.
 *
 ***************************************************************************/
-void ISR_RXCAN()
+void HAL_CAN_RxFifo0MsgPendingCallback(CAN_HandleTypeDef *hcan)
 {
+	HAL_GPIO_TogglePin(LD4_GPIO_Port, LD4_Pin);
+
+	CanRxMsgTypeDef rx;
+	//hcan1.pRxMsg = &rx;
```

Comment:
Why are these comments needed? We should not have commented code anywhere, especially without an explanation as to why it's there.